custom_components/octopus_intelligent/binary_sensor.py

```diff
@@ -44,7 +51,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             hass, 
             hass.data[DOMAIN][config_entry.entry_id][OCTOPUS_SYSTEM],
             "Octopus Intelligent Planned Dispatch Slot")
-    ], True)
+    ], False)  # False: data was already fetched by __init__.py async_setup_entry()
```

Comment:
This simple change from `True` to `False`, and similar changes in other files, is what delivers the subject of the PR — preventing a duplicate Octopus API call when the integration loads / Home Assistant starts.